docs/getting_started/run_schedule.md

```diff
@@ -50,7 +50,7 @@ object:
 from flytekit.remote import FlyteRemote
 from flytekit.configuration import Config
 
-remote = FlyteRemote(config=Config.auto())
+remote = FlyteRemote(config=Config.auto(),      default_project="flytesnacks", default_domain="development")
```

Comment:
@davidmirror-ops, this doesn't look right.

Reply:
Yep, sorry. I blame weird formatting on my editor.